blog/2023-11-02-netdata-plan-changes/index.md

```diff
@@ -0,0 +1,50 @@
+---
+slug: netdata-plan-changes
+title: "Upcoming changes to Netdata Cloud plans"
+description: "How will Netdata Cloud plans changes? What are the impacts and reasons."
+image: https://github.com/netdata/blog/assets/82235632/4f176516-17c7-42a5-9cab-51fad7184cdc
+authors: team
+tags: [systemd, systemd journal, logs, monitoring, security]
+keywords: [systemd journal, systemd, logs]
+---
+
+![plans](https://github.com/netdata/blog/assets/82235632/4f176516-17c7-42a5-9cab-51fad7184cdc)
+
+> **UPDATE**: On the **2023-11-08** Node and Dashboard limits will be applied on the Netdata Cloud **Community plan**, while all current features of the Community plan will remain the same.
+
+<!--truncate-->
+
+# What you need to know?
+
+* On Netdata Cloud Free Community plan, the number of **active nodes** that can be concurrently visualized on the Netdata dashboards, as well as the number of **active custom dashboards** for accounts created after 2023-11-06/07 will be subject to limits. 
+
+* Netdata Cloud accounts on the current Community or Early Bird plans created before **2023-11-07** will not be affected, and no limits will be set.
+
+* Any new Netdata Cloud user sign-up will be automatically enrolled to a **30-day Free Business Trial**, so all can experience Netdata Cloud without any limits or restrictions - no debit or credit cards will be required.
+
+* The changes above do not apply to our other plans: Netdata Cloud Pro, Business, On-prem subscriptions, and Netdata FOSS Agent versions.
```

Comment:
Should we talk about Pro explicitly in this statement as we are talking about moving them to Business?

Reply:
yeah probably makes sense to remove it